EKF/RingBuffer.h

```diff
@@ -130,7 +130,7 @@ class RingBuffer
 			int index = (_head - i);
 			index = index < 0 ? _size + index : index;
 
-			if (timestamp >= _buffer[index].time_us && timestamp - _buffer[index].time_us < 100000) {
+			if (timestamp >= _buffer[index].time_us && timestamp - _buffer[index].time_us < (int)1e5) {
```

Comment:
The timestamps I see are `uint64_t`, so we should cast to that.
And we need to be careful not to lose accuracy, but it's fine in these cases here.

Reply:
**I agree** because it's theoretically correct although it doesn't change the compile output in the online tool. I assume the compiler chooses the "bigger" of the two data types to do the comparison.

(And **if it would not**, there needs to be a >35 minutes time difference to break it. Would be a nasty bug to find.)